src/callbacks/amr.jl

```diff
@@ -315,46 +318,46 @@ end
 
 
 # TODO: Taal dimension agnostic
-# TODO: Taal refactor, merge the two loops of IndicatorThreeLevel and IndicatorLöhner etc.?
+# TODO: Taal refactor, merge the two loops of ControllerThreeLevel and IndicatorLöhner etc.?
 #       But that would remove the simplest possibility to write that stuff to a file...
 #       We could of course implement some additional logic and workarounds, but is it worth the effort?
-function (indicator::IndicatorThreeLevel)(u::AbstractArray{<:Any},
-                                          mesh::TreeMesh, equations, dg::DG, cache)
+function (controller::ControllerThreeLevel)(u::AbstractArray{<:Any},
```

Comment:
I wonder if it is really necessary here to make this a callable instead of a free function (I've left a similar comment elsewhere). We don't have to discuss it here (I'll put it on the next meeting agenda), but maybe for this specific case there's a strong rationale?